server/src/main/java/io/crate/protocols/postgres/PostgresWireProtocol.java

```diff
@@ -385,13 +386,25 @@ private void closeSession() {
         }
 
         @Override
-        public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
-            if (cause instanceof SocketException && cause.getMessage().equals("Connection reset")) {
+        public void exceptionCaught(ChannelHandlerContext ctx, Throwable t) throws Exception {
+            LOGGER.trace("Exception caught", t);
```

Comment:
Can move this into the `DecoderException block - the other two blocks already contain log statements so we'd end up with duplicate logging.

Reply:
I'll remove it, `Messages.sendErrorResponse` already trace logs the decode error.